app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -648,7 +645,8 @@ private void updatePlaybackSpeedButton() {
         if(butPlaybackSpeed == null) {
             return;
         }
-        if (controller == null) {
+        // TODO this possibly needs to change if we introduce an alternative button
+        if (controller == null || !(this instanceof AudioplayerActivity)) {
```

Comment:
shouldn't AudioplayerActivity override this method to support configuring the button?  The default implementation can just return without doing anything.